packages/sync/package.json

```diff
@@ -55,6 +56,7 @@
     "apollo-link-retry": "2.2.13",
     "apollo-link-ws": "1.0.17",
     "apollo-upload-client": "10.0.0",
+    "crypto-js": "^3.1.9-1",
```

Comment:
I think we can do better without doing extensive benchmarks:

https://www.npmjs.com/package/webcrypto

https://medium.com/@encryb/comparing-performance-of-javascript-cryptography-libraries-42fb138116f3